### PR TITLE
[sc-145174] Preferences copy updates

### PR DESCRIPTION
**Pull request by yusinto**
Cursor_and_Preferences

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ on:
   push:
     tags:
     branches:
-      - 2022 #TODO: Switch branch when strategy is finalized
```
Comment (yusinto) [inline .github/workflows/build.yml:20]: Unrelated, but this needs to be done.

**Review by carmenquan — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (jazanne) [reply .github/workflows/build.yml:20]: good catch, not sure how we missed that

File: src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt [hunk 2]
```diff
@@ -148,7 +148,7 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
                 } else {
                     DefaultComboBoxModel(arrayOf(defaultMessage))
                 }
-                row("Project") {
+                row("Project:") {
```
Comment (carmenquan) [inline src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt:151]: Victor had feedback in the blitz that Project is singular but Environments is plural. We should stick with one, maybe singular? @launchdarkly/docs-reviewers any thoughts

Comment (mmrj) [reply src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt:151]: I think singular, since you're selecting one of each.

Comment (jazanne) [reply src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt:151]: yea singular, and the Environments label is updated in https://github.com/launchdarkly/ld-intellij/pull/26

**Review by carmenquan — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (jazanne) [reply .github/workflows/build.yml:20]: fixed in another Pr ✅

**Pull request merged**